### PR TITLE
Add i18n for covid19

### PR DESCRIPTION
**Pull request by Gommorach**

### CODE REVIEW

File: assets/js/covid19.js [hunk 1]
```diff
@@ -106,6 +106,18 @@
             }
             return false;
         });
+
+        var browserLanguage = navigator.language.slice(0, 2);
```
Comment (simong) [inline assets/js/covid19.js]: In ally-ui's `TranslateUtil` we have:
```
const langs = navigator.languages;
            if (langs.length > 0) {
                return navigator.languages[0];
            } else if (NullUtil.hasValue(navigator.language)) {
                return navigator.language;
            } else {
                return null;
            }
```

I can't recall the semantics atm, but can you check whether we need to have a fallback like that here?

File: assets/js/covid19.js [hunk 2]
```diff
@@ -106,6 +106,18 @@
...
+        var language = ['de', 'en', 'es', 'fr', 'it'].indexOf(browserLanguage) > -1 ? browserLanguage : 'en';
+
+        $.i18n().load({
+            [language]: '/assets/locale/covid19/' + language +'.json'
```
Comment (simong) [inline assets/js/covid19.js]: Probs best to keep it simple for now and avoid any ECMA 6+ syntax.

File: assets/locale/covid19/es.json [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+	"NAVIGATION_LINK_RESOURCES_TITLE": "Recursos de Blackboard Ally",
+	"NAVIGATION_LINK_YOUTUBE": "YouTube",
+	"NAVIGATION_LINK_YOUTUBE_TITLE": "Blackboard Ally en YouTube",
+	"PRIVACY_POLICY": "Política de privacidad",
+	"TERMS_OF_USE": "Condiciones de uso",
+	"COPYRIGHT": "© Todos los derechos reservados Blackboard Inc"
+}
+
+
+
```
Comment (simong) [inline assets/locale/covid19/es.json]: 👀

File: assets/js/covid19.js [hunk 4]
```diff
@@ -106,6 +106,18 @@
             }
             return false;
         });
+
+        var browserLanguage = navigator.language.slice(0, 2);
+        var language = ['de', 'en', 'es', 'fr', 'it'].indexOf(browserLanguage) > -1 ? browserLanguage : 'en';
```
Comment (simong) [inline assets/js/covid19.js]: Can you also set the `lang` attribute of the `html` tag? That should cause the UI API to be localized as well. That might have to happen before the ally.ui.js script is invoked though.

File: assets/js/covid19.js (hunk 1 above)
Comment (Gommorach) [reply assets/js/covid19.js]: Frigging IE11. I add a similar fallback that should catch all options.

**Comment by simong** [comment]
👍

**Pull request merged**